src/web/components/KeyPairs/KeyPairsTable.scss

```diff
@@ -0,0 +1,65 @@
+.key-pairs {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+
+  .key-pairs-table {
+    margin-top: 28px;
+    font-size: 0.875rem;
+
+    thead {
+      font-weight: 700;
+      border: none;
+    }
+
+    .description {
+      width: 30%;
+    }
+
+    .subscription-id {
+      width: 10%;
+    }
+
+    .site-id {
+      width: 5%;
+    }
+
+    .public-key {
+      width: 40%;
```

Comment:
I wonder if we could add
```
overflow-wrap: anywhere;
```
Before:
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/ebae7397-d199-4818-a413-cd5e6b15eb94)

After:
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/84cf9023-1cb6-46bb-b474-0d192f9dda46)

Reply:
And maybe some sort of horizontal padding 
e.g. `padding-right: 10px`. Probably want that on all the `<th>`s

![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/675cef75-1d64-47b1-ac00-8a3b86637cee)

Reply:
Good ideas.  I think it looks a bit cleaner now:

![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/37344964/935f06c8-6d1d-4027-be56-507be633512e)